util/winutil/winutil.go

```diff
@@ -16,7 +16,10 @@ import (
 	"golang.org/x/sys/windows/registry"
 )
 
-const RegBase = `SOFTWARE\Tailscale IPN`
+const (
+	RegBase       = `SOFTWARE\Tailscale IPN`
```

Comment:
I feel like these should either both be exported or neither should be exported.

Reply:
but if they stay mixed, they shouldn't be in a const ( ... ) block together